cmd/nebula-cert/ca.go

```diff
@@ -122,6 +125,10 @@ func ca(args []string, out io.Writer, errOut io.Writer) error {
 		return fmt.Errorf("refusing to overwrite existing CA key: %s", *cf.outKeyPath)
 	}
 
+	if *cf.qr && *cf.outCertPath == "ca.crt" {
+		*cf.outCertPath = "ca.png"
+	}
```

Comment:
The way this works is very specific and not very intuitive.
Combining these flags in the wrong way will lead to unexpected results.

I would expect use of a `-qr` flag to produce the png in addition to the text-based cert.

Reply:
Yep. Totally agree again!